Postman Test Suite/Ed-Fi ODS-API Integration Test Suite ResponseTests.postman_collection.json

```diff
@@ -5553,18 +5608,16 @@
 													"});",
 													"",
 													"pm.test(\"Should indicate the source values are inconsistent\", () => {",
-													"  pm.expect(problemDetails.validationErrors).to.deep.include({",
-													"    \"$.gradingPeriodReference.schoolId\": [",
-													"      \"All values supplied for 'schoolId' must match. Review all references and align the following conflicting values: '777', '888'\"",
-													"    ],",
-													"    \"$.studentSectionAssociationReference.schoolId\": [",
-													"      \"All values supplied for 'schoolId' must match. Review all references and align the following conflicting values: '777', '888'\"",
-													"    ]",
-													"  });",
+													"  pm.expect(Object.keys(problemDetails.validationErrors).length).to.equal(2);",
```

Comment:
Converted these to use regular expression matching rather than "deep equals" matching to focus on the specific requirements (for these messages to be present), but not necessarily enforcing exact match behavior (like message order).